docs/guides/site_navigation/index.md

```diff
@@ -5,3 +5,7 @@ has_children: true
 parent: Guides
 nav_order: 5
 ---
+
```

Comment:
This index page can be empty.

The My Site > Header/Footer will be depreciated. Regardless, I overheard this morning you discuss with Paulina that this guidance is for theme developers not Creators so the fact they can edit the variable values in pages isn't necessary here either!